tests/run-tests

```diff
@@ -57,7 +65,7 @@ def run_micropython(pyb, args, test_file, is_special=False):
     had_crash = False
     if pyb is None:
         # run on PC
-        if test_file.startswith(('cmdline/', 'feature_check/')) or test_file in special_tests:
+        if test_file.startswith(('cmdline/', base_path('feature_check/'))) or test_file in special_tests:
```

Comment:
Why does `feature_check` need `base_path` but `cmdline` doesn't?

Reply:
Because test_file will still be a relative path in most cases, except not for feature_check tests. In `main()` tests still get listed relative to the test directory, also see my comment on this PR above to why that is, and I added this to the hlpe text as well.